cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go

```diff
@@ -176,7 +176,7 @@ func resetConfigDir(configPathDir string, dirsToClean []string, isDryRun bool) {
 			}
 		}
 	} else {
-		fmt.Printf("[reset] Would delete files: %v\n", filesToClean)
+		fmt.Printf("[dryrun] Would delete files: %v\n", filesToClean)
```

Comment:
Should file path be quoted using %q?

Reply:
filesToClean is a slice of strings, so that's why here is  %v